reference/divio-components-shell.rst

```diff
@@ -60,7 +60,7 @@ containers.
 **Processes** - each session is isolated from any other extant processes (web processes, workers,
 other shell sessions).
 
-**Caches** - if your site's cache relies on the database (the Divio Cloud Projects default) then
+**Caches** - if your site's cache relies on the database (the Divio Projects default) then
```

Comment:
_the Divio Projects default_ > _the default in Divio projects_

Reply:
Done